var/spack/repos/builtin/packages/pacifica-cli/package.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PacificaCli(PythonPackage):
+    """Python CLI for Pacifica Core Services"""
+
+    homepage = "https://github.com/pacifica/pacifica-cli/"
+
+    version('0.5.2', sha256='fee5fa8ac38ffec2e9199bff23afbbae697c4f7554f13e340104f8b20a62843f', url="https://pypi.io/packages/source/p/pacifica-cli/pacifica-cli-0.5.2.tar.gz")
+
+    depends_on('py-setuptools', type='build')
+    depends_on('py-wheel', type='build')
+    depends_on('py-importlib-metadata', type='run')
+    depends_on('py-jsonschema', type='run')
+    depends_on('py-requests', type='run')
+    depends_on('py-pacifica-uploader@0.3.1:0.3.9999', type='run')
+    depends_on('py-pacifica-downloader@0.4.1:0.4.9999', type='run')
+    depends_on('py-pacifica-namespace', type='run')
+    depends_on('py-pager', type='run')
```

Comment:
Python `install_requires` is equivalent to `type=('build', 'run')`, otherwise setuptools will try to download and install the dependencies itself, which won't work on an air-gapped network.

Reply:
i'm not sure what change you are asking for here, should I change them to be like this: `install_requires('py-pager')`

Reply:
You should change them to be like this:
```python
depends_on('py-pager', type=('build', 'run'))
```